h2o-core/src/jmh/java/water/NewChunkBench.java

```diff
@@ -0,0 +1,169 @@
+package water;
+
+import org.junit.Assert;
+import org.openjdk.jmh.annotations.*;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.profile.StackProfiler;
+import org.openjdk.jmh.runner.Runner;
+import org.openjdk.jmh.runner.RunnerException;
+import org.openjdk.jmh.runner.options.Options;
+import org.openjdk.jmh.runner.options.OptionsBuilder;
+import water.fvec.*;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Chunk access patterns benchmark
+ */
+@State(Scope.Thread)
+//@Fork(value = 1, jvmArgsAppend = "-XX:+PrintCompilation")
+@Fork(value = 1, jvmArgsAppend = "-Xmx12g")
+@Warmup(iterations = 5)
```

Comment:
Do you know that 5 is enough? Meaning did you check JIT, that after 5iterations, there is no heavy jitting?

Reply:
Yes, from my runs off my little laptop, the timing does not vary much.